homeassistant/components/huawei_lte/__init__.py

```diff
@@ -335,16 +336,19 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     def _connect() -> Connection:
         """Set up a connection."""
+        kwargs: dict[str, Any] = {
+            "timeout": CONNECTION_TIMEOUT,
+        }
+        if url.startswith("https://") and not entry.data.get(CONF_VERIFY_SSL):
```

Comment:
Before this PR, https was not supported.
With this PR, config entry data will always have `CONF_VERIFY_SSL` set.

Hence, can why do we use `get`?

Reply:
Good catch, addressed in d25b22448e818c4b8951fff1ac7d479967b0e3a8.